src/app/utils/mapLocations.js

```diff
@@ -0,0 +1,4 @@
+const mapLocations = locations => (locations ? locations.split(';') : [])
+  .map(location => (location === 'all' ? '' : location));
```

Comment:
Same comment as above. I forget what the reason for this choice was. It is a little confusing.

Reply:
same as above.